articles/what-api-endpoints-to-expose-to-the-public-internet.md

```diff
@@ -11,7 +11,7 @@ If you would like to manage hosts that can travel outside your VPN or intranet,
 
 ## Using Fleet Desktop on remote devices
 
-If you are using Fleet Desktop and want it to work on remote devices, the bare minimum API to expose is `/api/*/fleet/device/*/desktop`. This minimal endpoint will only provide the number of failing policies.
+If you're using Fleet Desktop `/api/*/fleet/device/*` and `/api/*/assets/*` must be exposed (for the end user **Fleet Desktop > My device** page).
```

Comment:
```suggestion
If you're using Fleet Desktop `/api/*/fleet/device/*` must be exposed in the API, and for the end user **Fleet Desktop > My device** page `/device/*` and `/assets/*` must be exposed.
```

Reply:
Looks good! Added those changes. Please feel free to merge if everything looks good 🚀